transformer/transformer.go

```diff
@@ -44,6 +44,7 @@ var transformerFunctionMap = map[string]func(*transformers.Context) error{
 	"absoluteurl":           transformers.AbsoluteURL,
 	"ampboilerplate":        transformers.AMPBoilerplate,
 	"ampruntimecss":         transformers.AMPRuntimeCSS,
+	"ampruntimejs":			 transformers.AMPRuntimeJS,
```

Comment:
Seems like a whitespace issue? Should be able to run `gofmt -w transformer/transformer.go` to fix.